src/api/uploads/BaseUpload.js

```diff
@@ -24,6 +24,8 @@ class BaseUpload extends Base {
 
     overwrite: boolean;
 
+    conflictCallback: Function;
```

Comment:
We've been trying to avoid generic types like Function. Can you add the input/output parameters?